conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -320,7 +321,7 @@ TEST_F(
   }
 
   if (domains.size() < 2) {
-    GTEST_SKIP() << "Skipping test due to not enough domains";
+    GTEST_SKIP() << "Skipping test due to insufficient domains";
```

Comment:
this does not seem consistent with the assert of line 91. Do we skip or assert when not enough domains are found in the system ? IMHO,  I think this is correct and not line 91: the number of domains is a HW feature, if the HW is not suitable for the test, then skip.

Reply:
Resolved, thank you.